tracer_manager/tracer_manager.F90

```diff
@@ -1072,7 +1081,14 @@ subroutine set_tracer_profile(model, n, tracer, err_msg)
     profile_type                   = 'Fixed'
     flag =parse(control,'surface_value',surf_value)
     multiplier = 1.0
-    tracer = surf_value
+    select type (tracer)
+    type is (real(kind=r4_kind))
+      tracer = surf_value
+    type is (real(kind=r8_kind))
+      tracer = surf_value
+    class default
+      call mpp_error(FATAL, "set_tracer_profile : unsupported kind")
```

Comment:
This error message should be more descriptive.  *"tracer is not one of the supported types of real(kind=4) or real(kind=8)."*